analysed-packages/dbus/version-1.14.6/README.md

```diff
@@ -0,0 +1,42 @@
+## Download Location
+
+https://dbus.freedesktop.org/releases/dbus/dbus-1.14.6.tar.xz
+
+## Package URL (purl)
+
+pkg:generic/dbus@1.14.6?download_url=https://dbus.freedesktop.org/releases/dbus/dbus-1.14.6.tar.xz
+
+## Creator
+
+Parian Golchin
+
+## Reviewers
+
+The information was reviewed by:
+
+* Mario Schuknecht
+* Sascha Teichmann
+* Oliver Fendt
+
+## Comment
+
+This package is dual licensed under GPL-2.0-or-later or
```

Comment:
It would be very nice if you can condense the different statements into one like:
This package is dual licensed GPL-2.0-or-later or AFL. To facilitate...
Please remove the MIT option these are only  a few files and not the entire package